build_projects/Microsoft.DotNet.Cli.Build.Framework/Command.cs

```diff
@@ -184,7 +184,6 @@ public CommandResult Execute()
             _process.EnableRaisingEvents = true;
 
             var sw = Stopwatch.StartNew();
-            ReportExecBegin();
```

Comment:
Ok. I'll move these to console.writeline for now. Reporter brings in a bunch of other dependencies that I don't want to maintain.